css/style.css

```diff
@@ -21,3 +21,9 @@ body{
 body.stopPageScroll{
 	overflow-y: hidden;
 }
+:root{
```

Comment:
move it to separete css file called "variables" and inlude like a first one before the rest